doc/api/index.md

```diff
@@ -1443,8 +1480,10 @@ player.setPreferredTextTracks([
 
 ---
 
-:warning: This method will have no effect for contents loaded in _DirectFile_
-mode (see [loadVideo options](./loadVideo_options.md#prop-transport)).
+:warning: This option will have no effect in _DirectFile_ mode
+(see [loadVideo options](./loadVideo_options.md#prop-transport)) when either :
+- No audio track API was supported on the current browser
```

Comment:
What the relation with the `audio track API`? :P

Reply:
done